lib/utils/__tests__/source_ip_check.test.ts

```diff
@@ -1,11 +1,10 @@
-import * as t from "io-ts";
-
 import { toAuthorizedCIDRs } from "../../models/service";
 import { ClientIp } from "../middlewares/client_ip_middleware";
 import { ResponseSuccessJson } from "../response";
 import { IPString } from "../strings";
 import { Tuple2 } from "../tuples";
 
+import { fromEither } from "fp-ts/lib/Option";
```

Comment:
I would import `Option` and call `Option.fromEither` below as `fromEither` alone is not really telling much, `Option.fromEither` will make it clear that we're converting an `Either` into an `Option`.

Btw I recall that `Either` had a `toOption` method, is it gone?

Reply:
yes it is